integration-testing/src/functionalTest/kotlin/kotlinx.atomicfu.gradle.plugin.test/cases/MppProjectTest.kt

```diff
@@ -41,4 +41,11 @@ class MppProjectTest {
         assert(mppSample.cleanAndBuild().isSuccessful)
         mppSample.checkConsumableDependencies()
     }
+    
+    @Test
+    fun testMppWasmBuild() {
+        assert(mppSample.cleanAndBuild().isSuccessful)
```

Comment:
I missed it during the integration tests review: please use `assertTrue` here and in other tests instead of `assert`. Behavior of the `assert` depends on runtime flags and for testing purposes the `kotlinx.test.assertTrue` is much better suited.

Reply:
Fixed